### PR TITLE
Python GAPIC: Speech v1

### PR DESCRIPTION
**Pull request by lukesneeringer**
This pull request was generated by artman. Please review it thoroughly before merging.

### CODE REVIEW

File: speech/google/cloud/gapic/speech/v1/speech_client_config.json [hunk 1]
```diff
@@ -6,9 +6,7 @@
           "DEADLINE_EXCEEDED",
           "UNAVAILABLE"
         ],
-        "non_idempotent": [
-          "UNAVAILABLE"
-        ]
+        "non_idempotent": []
```
Comment (tseaver) [inline speech/google/cloud/gapic/speech/v1/speech_client_config.json]: This bit is intentional?  Most of the others of these I have seen allow retries for `UNAVAILABLE`, even for non-idempotent methods.

Comment (lukesneeringer) [inline speech/google/cloud/gapic/speech/v1/speech_client_config.json]: Good question; I will find out. I _expect_ it is, but I am unsure.

Comment (lukesneeringer) [inline speech/google/cloud/gapic/speech/v1/speech_client_config.json]: @shinfan Any idea if this was an intentional change? As in, we _do not_ want to retry `UNAVAILABLE` in Speech?

Comment (lukesneeringer) [inline speech/google/cloud/gapic/speech/v1/speech_client_config.json]: I have not been able to chase this down, and this needs to go out today for a Monday announcement. I can make the change later if it ends up being wrong.

**Pull request merged**